scripts/slack.py

```
@@ -0,0 +1,75 @@
+    ).encode()
+
+    req = urllib.request.Request(
+        _SLACK_API,
+        data=data,
+        headers={
+            "Content-Type": "application/x-www-form-urlencoded",
+            "Cookie": f"d={d_cookie}",
+        },
+    )
+
+    with urllib.request.urlopen(req, timeout=30) as resp:
```

Ruff (0.15.15)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)